objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -1748,16 +1749,17 @@ public void testSuppressNullValuesHelper(boolean suppressNullValues) throws Inte
 
         // If suppressNullValues is set to true, we shouldn't have any results stored in our type
         if (suppressNullValues) {
-            assert responseBody.size() == 0;
-
-        // Otherwise, we should have some results and they should be empty arrays
+            assertEquals("Get responseBody content when none expected: " + responseBody, 
+                    0, responseBody.size());
         } else {
+            // Otherwise, we should have some results and they should be empty arrays
             assert responseBody.size() > 0;
 
             for (int i = 0; i < responseBody.size(); i++) {
                 JsonObject resultObject = (JsonObject) responseBody.get(i);
                 assert resultObject.get("results") instanceof JsonArray;
-                assert ((JsonArray) resultObject.get("results")).size() == 0;
+                assertEquals("Get resultObject.results content when none expected: " + responseBody,
+                        0, ((JsonArray) resultObject.get("results")).size());
```

Comment:
Sorry -- don't like _assertion failed_ result.  Need to know what went wrong, so opportunistically fixing them as appropriate.  Easy to miss once you're used to PowerAsserts in groovy.